recipe/conda_build_config.yaml

```diff
@@ -11,5 +11,5 @@ libclang_soversion:
 
 variant:
   - default
-  - root_63406  # [linux or osx]
+  - root_63602  # [linux or osx]
```

Comment:
Given that the current pinning has two root versions: if you want to also keep 6.34.x, we should do 6.34.8 (or whatever is the latest patch version) before 6.36.2

Reply:
It's not a general requirement. You can do whatever order you like. In principle we could create branches, but since we're _already_ talking about a maintenance branch (18.x), I don't want to multiply those by the number of root versions.

That means you just add the patches for whatever root version you want to build on top of whatever the state of the repo is, even if the minor version of root regresses.

We'll have to think how to handle the cling variants through all that (which generally get updated at the same time as root, so I'd like to keep this parallel in principle, but not a hard requirement)